subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/junit/result/TestReportDataCollector.java

```diff
@@ -70,6 +70,16 @@ public void afterSuite(TestDescriptor suite, TestResult result) {
             methodResult.completed(result);
             classResult.add(methodResult);
             results.put(suite.getName(), classResult);
+        } else if (result.getResultType() == TestResult.ResultType.SKIPPED && suite.getParent() != null) {
+            String parentName = suite.getParent().getDisplayName();
```

Comment:
Here you are using the fact that parametrized tests are treating as a suites?
A bit speculative, but seems like there is no another way to do it
Would you take a look also @donat ?

Reply:
I think this question is better suited for the @gradle/bt-core team.

Reply:
@ghale Thank you for the review & advice! Sorry for late response , pardon me if I misunderstand this😅, you mean we shouldn't report the entire skipped `Paramerized Test` as a whole, but every single one in that suite? E.g:
```
@disabled
@ParameterizedTest
@ValueSource(strings = ["First Name", "Second Name"])
interesting disabledParameterizedTest(testParam: String) {
  assertEquals(testParam, testParam)
}
```
-> in report:
```

| Test            | Method name                          | Duration | Result  |
|-----------------|--------------------------------------|----------|---------|
| [1] First Name  | disabledParameterizedTest(String)[1] | 0.027s   | ignored |
| [2] Second Name | disabledParameterizedTest(String)[2] | 0.001s   | ignored |

```

I'm not sure if it's necessary, for two reasons:
1. If we report each test case of a disabled parameterized tests, the report may look more verbose
2. To be fair enough, maven reports a disabled parameterized tests as one (see screenshot posted in the issue)

But I also agree to create a `ParameterizedTestSuiteDescriptor`, it improves readability in `TestReportDataCollector`

Reply:
No, my comment was simply about how to make the condition here more tight.  The original code was basically detecting a parameterized test by looking for a test suite with a parent.  This may be effective now, but it's brittle and it's basically just checking side effects.  I'm suggesting that we model this more tightly by detecting that it's a parameterized test suite during test execution (when we have more information) and passing something more explicit through to `TestReportDataCollector`.  So it would still show up as one skipped test, it's just that it's much more clear in `TestReportDataCollector` what the test suite is.

Reply:
@ghale Thanks! Really detailed guidance and steps, thanks for the help. I've updated it.

Reply:
Let's just remove the `&& suite.getParent() != null` here.